### PR TITLE
Fix Jq in Makefile and update requirements.txt

### PR DESCRIPTION
**Pull request by jsitu777**
**Which issue is resolved by this Pull Request:**
When running `make-install` there's error when intalling jq:
```
sudo apt-get install jq=1.5+dfsg-2 -y
E: Could not get lock /var/lib/dpkg/lock-frontend - open (11: Resource temporarily unavailable)
E: Unable to acquire the dpkg frontend lock (/var/lib/dpkg/lock-frontend), is another process using it?
Makefile:40: recipe for target 'install-jq' failed
make: *** [install-jq] Error 100

```

**Description of your changes:**
- update install jq command in Makefile
- update packages version in requirements.txt and verified there're no conflicting dependency versions

**Testing:**
- mannual testing by running `make-install` in a ubuntu ec2 instance

### CODE REVIEW

File: components/notebook-dockerfiles/pytorch/requirements.txt [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # These packages are installed for both python2 and python3
 
 # Jupyter related packages
-jupyter-console==6.4.4
-jupyterlab==3.4.4
-jupyterlab-git==0.38.0
-notebook==6.4.12
-ipykernel==6.15.1
+jupyter-console==6.6.3
```
Comment (surajkota) [inline components/notebook-dockerfiles/pytorch/requirements.txt:4]: revert? these should only be updated with notebook images are built

File: components/notebook-dockerfiles/tensorflow/requirements.txt [hunk 2]
```diff
@@ -1,15 +1,15 @@
 # These packages are installed for both python2 and python3
 
 # Jupyter related packages
-jupyter-console==6.4.4
-jupyterlab==3.4.4
-jupyterlab-git==0.38.0
-notebook==6.4.12
-ipykernel==6.15.1
+jupyter-console==6.6.3
```
Comment (surajkota) [inline components/notebook-dockerfiles/tensorflow/requirements.txt:4]: same revert

File: tests/e2e/requirements.txt [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
-PyYAML==5.4
-black==22.8.0
-mysql-connector-python==8.0.27
-retrying==1.3.3
-requests==2.28.1
-numpy==1.23.3
-sagemaker==2.0.0
-pandas==1.5.0
\ No newline at end of file
+boto3==1.26.117
```
Comment (surajkota) [inline tests/e2e/requirements.txt:1]: since this is being used by various scripts. Can we run one cognito-rds-s3 installation before merging?

**Review by surajkota — CHANGES_REQUESTED** [review]

File: components/notebook-dockerfiles/pytorch/requirements.txt (hunk 1 above)
Comment (jsitu777) [reply components/notebook-dockerfiles/pytorch/requirements.txt]: reverted

File: components/notebook-dockerfiles/tensorflow/requirements.txt (hunk 2 above)
Comment (jsitu777) [reply components/notebook-dockerfiles/tensorflow/requirements.txt]: reverted

File: tests/e2e/requirements.txt (hunk 3 above)
Comment (jsitu777) [reply tests/e2e/requirements.txt]: how about I revert it back to 1.18.50 first so we can merge this and I will test the new version offline.

Comment (surajkota) [reply tests/e2e/requirements.txt]: I meant the whole file, not just boto3

Comment (jsitu777) [reply tests/e2e/requirements.txt:1]: tested. all working

**Review by surajkota — APPROVED** [review]
thanks

**Pull request merged**